docs/build/environment/chopsticks.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 7
+---
+
+# Chopsticks & E2E Tests
+
+:::note
+Create parallel realities of our Substrate networks.
+:::
+
+Forking a live blockchain at any block with the corresponding memory and state offers several benefits for developers, users, and the blockchain ecosystem as a whole. This capability allows for more accurate testing of transactions and upgrades, which can lead to better decision-making and more robust blockchain networks. Key benefits include:
+
+1. Improved security: By forking a live blockchain, developers can test proposed changes, identify vulnerabilities, and address potential security threats in a controlled environment, reducing the risk of attacks on the main network.
+
+2. Enhanced performance: Testing transactions and upgrades on a forked version of the blockchain enables developers to analyze the performance implications of their changes, such as transaction throughput and latency. This can lead to optimizations that enhance the overall performance of the main blockchain network.
+
+3. Reduced downtime: Forking a live blockchain for testing purposes can help minimize network downtime during upgrades or other maintenance activities, as it allows developers to thoroughly test and validate changes before implementation on the main network.
+
+4. Increased user confidence: Users can feel more confident in the stability and reliability of the blockchain network when they know that proposed changes have been rigorously tested in an environment that closely mirrors the live network.
+
+5. Facilitated innovation: Forking a live blockchain provides a space for experimentation, allowing developers to test new features, protocols, and consensus algorithms without disrupting the main network. This fosters innovation and can lead to the development of novel applications and use cases for blockchain technology.
```

Comment:
Also, I would not overpromise all of these statements - Lets leave this to Chopsticks repo to promise, lets keep the benefits to tehnical ones - as this is builders technical documentation

Reply:
I've removed the optimistic sentiment sentences.